crates/sui-protocol-config/src/lib.rs

```diff
@@ -292,6 +294,15 @@ struct FeatureFlags {
     // Enable receiving sent objects
     #[serde(skip_serializing_if = "is_false")]
     receive_objects: bool,
+
+    // Enable throughput aware consensus submission
+    #[serde(skip_serializing_if = "is_false")]
+    throughput_aware_consensus_submission: bool,
+
+    // The throughput profile ranges. The range values are represented as an array and are index 0
```

Comment:
IIUC there will only be two profiles (low vs high), and only one boundary is needed? Would the first element always be 0?

Reply:
Exactly, unfortunately the config is not pretty, but for completeness decided to always make it define the Low setting. I am thinking of changing a bit. 

Regarding the profile levels I am thinking two iterations to understand and define the levels. To achieve that I'll need to do some benchmark testing in testnet.

**Setup 1**
Simulate pretty much what we have today and compare to past behaviour.

* Two levels only, `Low = 0, High = 2000` 
* For `profile = Low` the `LOW_THROUGHPUT_LATENCY_MS = 1250`

benchmark it for traffic between 0 - 5K tps and see how the network behaves.

**Setup 2**
* Three levels, `Low = 0, Medium = 500, High = 2000`
* The latency levels should accordingly be:
->  `profile = Low, LOW_THROUGHPUT_LATENCY_MS = 0` (basically 1st & 2nd validator co-submit)
->  `profile = Medium, LOW_THROUGHPUT_LATENCY_MS = 1250`
->  `profile = High, default latency from latency observer`

benchmark it for traffic between 0 - 5K tps and see how the network behaves.

please let me know your thoughts on this.